views/blog.erb

```diff
@@ -0,0 +1,85 @@
+<html>
+<header>
+  <h1>My Blog*:</h1>
+  <p>
+    Except not really because I hate blogs and can't maintain them.
+  </p>
+</header>
+
+<body>
+  <nav>
+  <ul>
+    <li><a href="./projects.erb">Projects</a></li>
+    <li><a href="./about.erb">About</a></li>
+    <li><a href="./index.erb">Home</a></li>
+  </ul>
+  </nav>
+
+  <h2>My life in snippets:</h2>
+  <section>
+    <h3>Things I ate today</h3>
+    <p>
+      10.25.2015
+    </p>
+    <ol>
+      <li>
+        Breakfast Taco
+      </li>
+      <li>
+        Bean Taco (2)
+      </li>
+      <li>
+        Tofu Tacos (with extra cilantro & spice)
+      </li>
+    </ol>
+  </section>
+
+  <section>
+    <h3>How I found my son</h3>
+    <p>
+      12.5.2002
+    </p>
+
+      <p>
+        <img src = "clur.jpg" alt = "Claire" />
+      </p>
+
+    <p>
+      Life along the Great Barrier Reef is full of dangers when you are a tiny clown fish like me. There I was, a single parent determined to protect my only son Nemo, and there are constant fears and anxieties. When, on his first day of school, Nemo defies me  and swam off alone to investigate a boat, he was suddenly scooped up by a diver as I helplessly watched.
+    </p>
+    <p>
+      I desperately swam off in search of my son. As I passed a school of fish I bumped into Dory, an agreeable blue tang with severe short-term memory loss, who offered to help. She was tiresome but some excellent comic relief. Together we set out on an impossible mission, finding ourselves in troubled waters and contending with such hazards as sharks, deadly angler fishes and a forest of jellyfish.
+    </p>
+    <p>
+      Meanwhile, in a dentist's office overlooking Sydney Harbor, Nemo landed in a fish tank that is home to a colorful group of characters who want to initiate him into their gang. When word of Marlin and Dory's ocean adventures gets back to Nemo, he was so thrilled to learn of my search for him that he became motivated to move forward with a daring escape plan of his own.
+    </p>
+
+    <p>
+      Watch my vlog about the whole experience <a href="https://www.youtube.com/watch?v=wZdpNglLbt8">here!</a>
+    </p>
+  </section>
+
+  <section>
+    <h3>What is this?!</h3>
+    <p>
+      6.29.1990
+    </p>
+    <p>
+      AHHHHHHHHHHH what is this world?! It's so bright! Why is everyone congratulating my carrier? I was so happy for 9 months why did everything have to change? I'm hungry! Wahh!
+    </p>
+  </section>
+
+
+  <aside>
+    This is about the frequency in which you should expect blog posts from me
+  </aside>
+</body>
+
+
+<footer>
+   <p>
+     If you blog too much, you never have time to hang out with me.
+   </p>
+
```

Comment:
Watch your extra whitespace - you only need one line of whitespace to separate out blocks of HTML